.circleci/config.yml

```diff
@@ -859,29 +610,26 @@ jobs:
 
   test-downgrade:
     executor: cross-builder
-    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
     steps:
       - checkout
       - attach_workspace:
-          at: /home/circleci/go/src/github.com/influxdata/influxdb
+          at: .
       - run:
           name: Run downgrade tests
           command: |
-            ./scripts/ci/test-downgrade.sh $(pwd)/dist/influxd_linux_amd64/influxd
+            ./scripts/ci/test-downgrade.sh $(pwd)/bin/influxd_linux_amd64/influxd
 
   share-testing-image:
-    executor: linux-amd64
-    working_directory: /home/circleci/go/src/github.com/influxdata/influxdb
+    docker:
+      - image: cimg/base:stable
     steps:
-      - attach_workspace:
-          at: /home/circleci/go/src/github.com/influxdata/influxdb
       - quay_login
       - run:
           name: Push the image to Quay
           command: |
-            docker load < docker-image.tar
             docker tag quay.io/influxdb/oss-acceptance:${CIRCLE_SHA1} quay.io/influxdb/oss-acceptance:latest
             docker push quay.io/influxdb/oss-acceptance:latest
+
```

Comment:
`VERSION="v$BRANCH"` makes the changelog `vmaster`, which is not great but is existing.

Reply:
@wbaker85 fyi